### ISSUE TITLE
Minor issue: Nutscript help tab won't display any subcategories.

### ISSUE THREAD

**Issue by daxxom** [Chessnut/NutScript#641]
^ This. I can't see the help tab with commands, flags, and plugins, but trying to click on a category yields no response and the page stays the same. 

Could someone explain to me what I did to mess this up?

**Comment by Myaats** [comment]
Is this a fresh version of NutScript and are you getting some LUA errors client or serverside?

**Comment by daxxom** [comment]
By fresh do you mean clean version that's just been reinstalled? If so no, And I here are my errors, not sure if they're server side or client side. (not sure how to determine either.)

I'm not sure how to read these very well, I'm very beginner. Apologies for using the Issues forum so much, I just cant find another way to learn N.S without asking for help.

[ERROR] gamemodes/fallout/schema/sh_config.lua:17: table index is nil
1. unknown - gamemodes/fallout/schema/sh_config.lua:17
   1. include - [C]:-1
   2. include - gamemodes/nutscript/gamemode/core/sh_util.lua:17
      1. unknown - gamemodes/fallout/schema/sh_schema.lua:131
      2. include - [C]:-1
      3. include - gamemodes/nutscript/gamemode/core/sh_util.lua:17
      4. load - gamemodes/nutscript/gamemode/core/libs/sh_plugin.lua:44
         1. initialize - gamemodes/nutscript/gamemode/core/libs/sh_plugin.lua:163
         2. unknown - gamemodes/nutscript/gamemode/shared.lua:65

[ERROR] gamemodes/fallout/schema/sh_schema.lua:170: bad argument #1 to 'pairs' (table expected, got nil)
1. pairs - [C]:-1
   1. unknown - gamemodes/fallout/schema/sh_schema.lua:170
   2. include - [C]:-1
      1. include - gamemodes/nutscript/gamemode/core/sh_util.lua:17
      2. load - gamemodes/nutscript/gamemode/core/libs/sh_plugin.lua:44
      3. initialize - gamemodes/nutscript/gamemode/core/libs/sh_plugin.lua:163
      4. unknown - gamemodes/nutscript/gamemode/shared.lua:65

Note: I'm using the default HL2 schema and re purposing it for something else.

**Comment by Chessnut** [comment]
Well did you keep the faction variable names?

**Comment by daxxom** [comment]
I did, I should go remove them, But it seemed different than the issue I was having, Reintalling the NS schema file seemed to fix my current issue about the help tab not loading it's contents- but It seems still have some others. For instance: The command /charsearch when typed does nothing, doesn't say that I attempted to run the command and have failed one of the inputs, or bring up another players inventory. As well as my currency won't change it's title despite me renaming it in "nut.currency.set("", "Name", "_Nameplural_")"

Where did I go wrong? As far as I can tell the console isn't showing me any errors for those line in initialization and when I try to execute them.

Thanks.
